.vscode/cspell.json

```diff
@@ -469,4 +470,4 @@
     }
   ],
   "allowCompoundWords": true
-}
+}
```

Comment:
nit: add new line at eof so we don't get this warning on GitHub

Reply:
Added